src/main/kotlin/net/atos/zac/shared/model/Paging.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.shared.model
+
+class Paging(val page: Int, val maxResults: Int) {
+    val firstResult: Int
+        get() = page * maxResults
```

Comment:
I always get this of these `get()`'s in the converted Kotlin code and replace it with a normal `getFirstResult()` etc function because I think that is much more readable.